shell_integration/dolphin/ownclouddolphinpluginhelper.cpp

```diff
@@ -125,17 +124,19 @@ void OwncloudDolphinPluginHelper::slotReadyRead()
                 return;
             }
         } else if (line.startsWith("V2/GET_CLIENT_ICON_RESULT:")) {
-            line.remove(0, QString("V2/GET_CLIENT_ICON_RESULT:").size());
+            line.remove(0, QStringLiteral("V2/GET_CLIENT_ICON_RESULT:").size());
```

Comment:
Hm maybe we could split at the : before the else if and only compare the command (`command:argumen`)
https://github.com/owncloud/client/blob/master/src/gui/socketapi/socketapi.cpp#L354